src/pages/workspaces/workspace/analysis/AppLauncher.js

```diff
@@ -27,13 +27,14 @@ const ApplicationLauncher = _.flow(
   })
 )(({
   name: workspaceName, sparkInterface, analysesData: { runtimes, refreshRuntimes },
-  application, workspace: { azureContext, workspace: { namespace, name, workspaceId, googleProject, bucketName } }
+  application, workspace: { azureContext, workspace: { workspaceId, googleProject, bucketName } }
 }, _ref) => {
-  const [busy, setBusy] = useState(true)
+  const [busy, setBusy] = useState(false)
```

Comment:
why are we removing the initial loading spinner? I'm not sure how exactly this PR accomplishes the goal of the ticket mechanically. It seems like we are just removing the initial spinner that awaits the initial runtime load and check for locks.

Reply:
I struggled to find anything that seemed to be a "callback" for when Jupyter was ready.

This approach is fairly simple, and will display a message for as long as possible before the first JupyterLab render.

Reply:
What causes it to disappear when Jupyter lab is actually running? Seems like it will always be displayed while the app is running?

Reply:
Once JupyterLab renders, the iframe has opacity so the loading spinner and message are hidden behind it via zIndex.

Reply:
ah ok. That means the iframe isn't inserted into the DOM/doesn't have a size until the `src` http request has been resolved I guess. A bit of an indirect way to do it, maybe a comment is warranted, but it gets the job done!